src/Processors/AugmentProperties.php

```diff
@@ -113,4 +112,26 @@ public function __invoke(Analysis $analysis)
             }
         }
     }
+
+    /**
+     * @param string $type
```

Comment:
the phpdoc does not add anything usefull and does not have the correct var name. imo you can remove the phpdoc

Reply:
Ok, I will update the pr.